### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by quentinmananes**

### CODE REVIEW

File: src/HttpRequest.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import requests
+
+
+class HttpRequest:
+    def __init__(self):
+        self.headers = {}
+        self.baseUrl = "http://schood.fr:8080"
```
Comment (Exiels) [inline src/HttpRequest.py:7]: Https

File: src/HttpRequest.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        return requests.post(self.baseUrl + route, headers=self.headers, data=data)
+
+    def patch(self, route, data):
+        return requests.patch(self.baseUrl + route, headers=self.headers, data=data)
+
+    def delete(self, route):
+        return requests.delete(self.baseUrl + route, headers=self.headers)
+
+
+httpRequest = HttpRequest()
```
Comment (Exiels) [inline src/HttpRequest.py:25]: Does this handle https ?

File: src/User.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+class User:
+    def __init__(self):
+        self.token = None
+        self._id = None
+        self.firstName = None
+        self.lastName = None
+        self.email = None
+        self.role = None
+        self.classes = None
+        self.picture = None
```
Comment (Exiels) [inline src/User.py:13]: Maybe try to store it globally to later use ?

**Review by Exiels — CHANGES_REQUESTED** [review]

File: src/HttpRequest.py (hunk 2 above)
Comment (quentinmananes) [reply src/HttpRequest.py:25]: Yes

File: src/User.py (hunk 3 above)
Comment (quentinmananes) [reply src/User.py:13]: Was already done at the bottom of the file, but extracted it with other global variables to a specific file 'utils/globalVars.py'

**Review by Exiels — APPROVED** [review]

**Pull request merged**